.zuul/playbooks/envoy-build/run.yaml

```diff
@@ -0,0 +1,36 @@
+- hosts: all
+  become: yes
+  tasks:
+    - name: Build envoy
+      shell:
+        cmd: |
+          apt update
+          apt-get install -y \
+             libtool \
+             cmake \
+             automake \
+             autoconf \
+             make \
+             ninja-build \
+             curl \
+             unzip \
+             virtualenv 
+             
+          apt-get install -y software-properties-common
+          add-apt-repository -y ppa:ubuntu-toolchain-r/test
+          apt update
+          apt install g++-7 -y
+          
+          update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 60 \
+                                   --slave /usr/bin/g++ g++ /usr/bin/g++-7 
+          update-alternatives --config gcc
+          gcc --version
+          g++ --version
+          
+          git clone https://github.com/envoyproxy/envoy.git
```

Comment:
See my comments in https://github.com/theopenlab/containerd/pull/1#pullrequestreview-225350266

Reply:
done